README.md

```diff
@@ -8,6 +8,7 @@ It is also helpful for local development to start multiple commands easily with
 
 - [robmuxinator](#robmuxinator)
   - [Installation](#installation)
+    - [Nix](#nix)
```

Comment:
I will add an additional headline for `pip` for consistency

Reply:
Add `Pip` entry.